_gsocproposals/2021/proposal_rucio_cernbox.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Rucio and CS3APIs: enabling data management for the ScienceMesh cloud
```

Comment:
You cannot have a ':' in your title: it is confusing for jekyll…

Reply:
o.O